scripts/basic-workflow-tests.sh

```diff
@@ -216,6 +217,15 @@ test_exec() {
     step Kill ilab model serve $PID
     kill $PID
 
+    if [ "$SKIP_TRAIN" -eq 1 ]; then
+        # TODO - Drop this later.
+        # This is only a temporary measure while we bootstrap different CI workflows.
+        # There are some larger environments where it only makes sense to test the new
+        # training workflow using the training library, but that is not yet integrated
+        # here. Skip training for those environments for now.
+        return
```

Comment:
Actually, would you mind logging that we return here? (add some value for debugging I guess...)

Reply:
added a msg, thanks!